Library/Homebrew/extend/os/linux/extend/ENV/std.rb

```diff
@@ -2,10 +2,18 @@
 # frozen_string_literal: true
 
 module Stdenv
-  def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_arch: nil, testing_formula: false)
+  def setup_build_environment(
```

Comment:
Keep these in as few lines as rubocop will allow (and same to all similar cases below).

Reply:
Done in cd9fe97c551d4e9b5f0a702a1382a6ccbe494773